packages/gatsby-theme-newrelic/src/components/Link.js

```diff
@@ -29,19 +35,26 @@ const Link = ({ to, ...props }) => {
   }
 
   if (isHash(to)) {
-    // eslint-disable-next-line jsx-a11y/anchor-has-content
     return <a href={to} {...props} />;
   }
 
   if (isExternal(to)) {
-    // eslint-disable-next-line jsx-a11y/anchor-has-content
-    return <a href={to} target="_blank" rel="noopener noreferrer" {...props} />;
+    return (
```

Comment:
Id there a reason why we're not using the `<ExternalLink />` component here?

Reply:
I want to deprecate `ExternalLink` and have everything go through `Link`, since this component is much smarter and embeds the same functionality. Also makes it easy to implement since you can use 1 component for all links. If anything, I'd go the other way around and make `ExternalLink` use `Link` under the hood, but again, I'd rather just deprecate that component altogether.